plaidml2/op/op.h

```diff
@@ -107,6 +107,47 @@ inline edsl::Tensor prod(const edsl::Tensor& I, const edsl::Value& axes = edsl::
   return details::op("prod", args).as_tensor();
 }
 
+class Relu {
+ protected:
+  edsl::Tensor I_;
+  edsl::Tensor alpha_ = edsl::None().as_tensor();
```

Comment:
I'm not sure this is actually a valid thing to do? Probably need to use a default `Tensor::Tensor()` constructor.

Reply:
Actually, since it has a default constructor, no need to have an `= rhs`